ocs_ingester/utils/fits.py

```diff
@@ -29,6 +32,19 @@ def get_from_start(self):
         self.fileobj.seek(0)
         return self.fileobj
 
+    def is_valid_fits(self):
+        """
+        Use astropy to determine if a file is a valid FITS file
+        :return: True if valid FITS, False if not
+        """
+        try:
+            with self.get_fits() as fits_file:
+                hdu_list = fits.open(io.BytesIO(fits_file.read()), mode='readonly')
```

Comment:
Do you have to call a close method on hdu_list?

Reply:
I am not sure - will investigate. It'd probably be safe to. I use the `get_fits` context manager which closes the file object, but I'll check on the hdu_list.

Reply:
I went ahead and replicated what was done [here]( https://github.com/observatorycontrolsystem/ingester/blob/9281b5552474ecd09d163e4e3fbc607578f86003/ocs_ingester/fits.py#L40), by using `fits.open` as a context manager. I think it's a safer bet. Thanks for finding that!